src/main/java/com/github/fge/jsonpatch/diff/DiffProcessor.java

```diff
@@ -26,12 +26,9 @@
 import com.github.fge.jsonpatch.JsonPatchOperation;
 import com.google.common.base.Equivalence;
```

Comment:
So `Equivalence` and `Predicate` still need to be removed. I've patched and synched through and I still get these errors. I'll take few minutes to see if they're easily removable.

Reply:
Done. See #66